src/rgw/rgw_dedup_store.cc

```diff
@@ -0,0 +1,830 @@
+#include "include/rados/rados_types.hpp"
```

Comment:
@adamemerson would appriciate your help reviewing rgw_dedup_store.cc/h
this is the code that does the I/O for the dedup module.